src/mesh/generated/meshtastic/mesh.pb.h

```diff
@@ -189,13 +189,19 @@ typedef enum _meshtastic_HardwareModel {
     meshtastic_HardwareModel_RADIOMASTER_900_BANDIT = 74,
     /* Minewsemi ME25LS01 (ME25LE01_V1.0). NRF52840 w/ LR1110 radio, buttons and leds and pins. */
     meshtastic_HardwareModel_ME25LS01_4Y10TD = 75,
+  
+
     /* RP2040_FEATHER_RFM95
  Adafruit Feather RP2040 with RFM95 LoRa Radio RFM95 with SX1272, SSD1306 OLED
  https://www.adafruit.com/product/5714
  https://www.adafruit.com/product/326
  https://www.adafruit.com/product/938
   ^^^ short A0 to switch to I2C address 0x3C */
     meshtastic_HardwareModel_RP2040_FEATHER_RFM95 = 76,
+    /* M5 esp32 based MCU modules with enclosure, TFT and LORA Shields. All Variants (Basic, Core, Fire, Core2, Paper) https://m5stack.com/ */
+    meshtastic_HardwareModel_M5STACK_COREBASIC=77,
+    /* M5 esp32 based MCU modules with enclosure, TFT and LORA Shields. All Variants (Basic, Core, Fire, Core2, Paper) https://m5stack.com/ */
+    meshtastic_HardwareModel_M5STACK_CORE2=78,
     /* ------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
These are generated from the protobuf repository. Changes here are overwritten and need to be put as a PR there instead.

Reply:
I have applied at protobufs